> Fuuu. Tests have been running but not failing the build?

Yup:

```
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_build_body
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_build_get_hook
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_check_session_id
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_delete_batch_fail
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_delete_batch_success
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_delete_batch_validation
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_get_batch_fail
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_get_batch_state_fail
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_get_batch_state_missing
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_get_batch_state_success
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_get_batch_state_validation
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_get_batch_success
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_get_batch_validation
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_invalid_uri
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_missing_batch_id
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_missing_host
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_parameters_validation
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_parse_post_response
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_post_batch_arguments
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_post_batch_fail
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_post_batch_success
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_validate_extra_conf
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_validate_list_of_stringables
ERROR tests/providers/apache/livy/hooks/test_livy.py::TestLivyHook::test_validate_size_format
FAILED tests/executors/test_dask_executor.py::TestDaskExecutor::test_dask_executor_functions
FAILED tests/executors/test_dask_executor.py::TestDaskExecutorTLS::test_tls
FAILED tests/executors/test_kubernetes_executor.py::TestKubernetesExecutor::test_run_next_exception
FAILED tests/executors/test_local_executor.py::TestLocalExecutor::test_execution_limited_parallelism
FAILED tests/executors/test_local_executor.py::TestLocalExecutor::test_execution_unlimited_parallelism

FAILED tests/executors/test_local_executor.py::TestLocalExecutor::test_execution_limited_parallelism
FAILED tests/executors/test_local_executor.py::TestLocalExecutor::test_execution_unlimited_parallelism
FAILED tests/models/test_dagcode.py::TestDagCode::test_remove_unused_code - A...
FAILED tests/providers/amazon/aws/operators/test_google_api_to_s3_transfer.py::TestGoogleApiToS3Transfer::test_execute
FAILED tests/providers/amazon/aws/operators/test_google_api_to_s3_transfer.py::TestGoogleApiToS3Transfer::test_execute_with_xcom
FAILED tests/providers/amazon/aws/operators/test_google_api_to_s3_transfer.py::TestGoogleApiToS3Transfer::test_execute_with_xcom_exceeded_max_xcom_size
FAILED tests/providers/discord/hooks/test_discord_webhook.py::TestDiscordWebhookHook::test_build_discord_payload
FAILED tests/providers/discord/hooks/test_discord_webhook.py::TestDiscordWebhookHook::test_build_discord_payload_message_length
FAILED tests/providers/discord/hooks/test_discord_webhook.py::TestDiscordWebhookHook::test_get_webhook_endpoint_conn_id
FAILED tests/providers/discord/hooks/test_discord_webhook.py::TestDiscordWebhookHook::test_get_webhook_endpoint_invalid_url
FAILED tests/providers/discord/hooks/test_discord_webhook.py::TestDiscordWebhookHook::test_get_webhook_endpoint_manual_token
FAILED tests/providers/google/common/hooks/test_discovery_api.py::TestGoogleDiscoveryApiHook::test_get_conn
FAILED tests/providers/google/common/hooks/test_discovery_api.py::TestGoogleDiscoveryApiHook::test_query
FAILED tests/providers/google/common/hooks/test_discovery_api.py::TestGoogleDiscoveryApiHook::test_query_with_pagination
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_connect_and_disconnect
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_found
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_not_found
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_with_escaping_chars
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_with_latest_only
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_with_mail_filter
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_with_regex_found
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_with_regex_not_found
FAILED tests/providers/imap/hooks/test_imap.py::TestImapHook::test_download_mail_attachments_with_symlink
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 50 failures !!!!!!!!!!!!!!!!!!!!!!!!!!
= 26 failed, 4521 passed, 338 skipped, 1 xpassed, 9 warnings, 24 errors in 623.63s (0:10:23) =
```

--- a/scripts/ci/ci_run_airflow_testing.sh
+++ b/scripts/ci/ci_run_airflow_testing.sh
@@ -106,25 +106,4 @@ done
 
 RUN_INTEGRATION_TESTS=${RUN_INTEGRATION_TESTS:=""}
 
-# Repeat tests in case initialization failed
-set +u
-MAX_RETRIES=4
-while [[ ${MAX_RETRIES} -ge "0" ]]
-do
-    set +e
-    run_airflow_testing_in_docker "${@}"
-    EXIT_CODE=$?
-    set -e
-    if [[ ${EXIT_CODE} == 254 ]] ; then
-        echo
-        echo "Retrying on initialization failure. Remaining ${MAX_RETRIES} retries left"
-        echo
-        MAX_RETRIES=$(( MAX_RETRIES - 1 ))
-        # Cleanup docker containers and images to make sure everything is retried from scratch
-        docker-compose -f "${MY_DIR}/docker-compose/base.yml" down --remove-orphans --timeout 20
-        docker system prune --force --volumes
-        continue
-    fi
-    exit ${EXIT_CODE}
-done
-set -u
+run_airflow_testing_in_docker "${@}"

